packages/sources/enzyme/README.md

```diff
@@ -0,0 +1,96 @@
+# Chainlink External Adapter for Enzyme
+
+Adapter to interact with Enzyme contracts.
+
+### Environment Variables
```

Comment:
Can be `##`

Reply:
This is from our template, we can leave as-is.